test/cli/template-delete/complex/in/main/extractor/keboola.ex-db-mysql/with-rows/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "parameters": {
     "db": {
-      "#password": "KBC::ProjectSecure::%s",
+      "#password": "KBC::ProjectSecure%s",
```

Comment:
Pridal som si medzi lokalne testovacie projekte jeden projekt, a zacalo mi to padat.
Namiesto `KBC::ProjectSecure::` sa tam generuje `KBC::ProjectSecureKV::`, mozno ten projekt nie je zmigrovany na novu Queue, alebo nieco podobne ...

Radsej som to ale upravil v testoch.

`KBC::ProjectSecureKV` je validny prefix, mame to aj v kode:
https://github.com/keboola/go-client/blob/a634d84e231793863bf95bbff3c8165ba92569e9/pkg/encryptionapi/utils.go#L15